Updated Pong

JoeNeglia opened the pull request:
Still not perfect, but the AI paddle actually tracks the ball. I tested it on my local environment and it worked fine. 
- added spin
- added different speeds
- everything can be adjusted accordingly

JoeNeglia committed:
Updated Pong

JoeNeglia committed:
Merge branch 'Dev-6' into Joe-127

Kataruse reviewed (approved):
GREEN
This is sick man!

T0xen reviewed (commented):
YELLOW
Functionally works a lot better than the previous version but the canvas is a different size now compared to the old one, which might be breaking the constraints for the game

JoeNeglia committed:
Fixed Canvas

Updated canvas size to fixed dimensions and adjusted game parameters for paddle and ball speed.

JoeNeglia committed:
Merge branch 'Dev-6' into Joe-127

Andersonjb1 committed:
Merge branch 'Dev-6' into Joe-127

Andersonjb1 reviewed (approved):
# Green
